unstract/connectors/src/unstract/connectors/filesystems/unstract_file_system.py

```
@@ -103,6 +103,116 @@ def is_dir_by_metadata(self, metadata: dict[str, Any]) -> bool:
+            if date.tzinfo is None:
+                # Naive datetime - assume UTC
+                return date.replace(tzinfo=UTC)
+            else:
+                # Convert to UTC
+                return date.astimezone(UTC)
+
+        try:
+            return sorted(
+                file_metadata_list, key=get_modified_date, reverse=not ascending
+            )
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)